bin/p2-preparer/main.go

```diff
@@ -85,6 +90,44 @@ func main() {
 		logger.WithError(err).Fatalf("Could not do initial build reality at launch: %s", err)
 	}
 
+	whiteListPods := make(map[types.PodID]bool)
+	// Keep loopinng the white list of pods in pod whitelist file, and install the non existing pods
+	// exit while the white list is empty
+	if preparerConfig.PodWhitelistFile != "" {
+		for {
+			_, err := os.Stat(preparerConfig.PodWhitelistFile)
+			if os.IsNotExist(err) {
+				logger.WithError(err).Warningf("Pod whilelist path is not valid")
```

Comment:
Is a warning the most semantically correct response to this error? A warning allows continued execution. The preparer will still attempt to read the file that we have just verified does not exist.